docs-content/sdk/go.md

```diff
@@ -20,10 +20,20 @@ slug: go
   <div className="left">
     <h3>highlight.Start()</h3>
     <p>Starts the background goroutine for transmitting metrics and errors.</p>
+    <h6>Options</h6>
+    <aside className="parameter">
+      <h5><code>WithServiceName</code> <code>optional</code></h5>
+      <p>The name of your app.</p>
+      <h5><code>WithServiceVersion</code> <code>optional</code></h5>
+      <p>The version of this app. We recommend setting this to the most recent deploy SHA of your app.</p>
+    </aside>
   </div>
   <div className="right">
     <code>
-        highlight.Start()
+        highlight.Start(
+          highlight.WithServiceName("my-app"),
+          highlight.WithServiceVersion("1.0.0"),
```

Comment:
Should we call out something like a git SHA here instead of a number version? I saw the callout above, but thinking people may skip over and just think they should use a number.

[Same for below]